e2e/cypress/integration/interactive_dialog/simple_dialog_spec.js

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
+*/
+
+const webhookUtils = require('../../../utils/webhook_utils');
+
+let createdCommand;
+let simpleDialog;
+
+describe('ID17212 Interactive Dialog without element', () => {
+    before(() => {
+        // Set required ServiceSettings
+        const newSettings = {
+            ServiceSettings: {
+                AllowedUntrustedInternalConnections: 'localhost',
+                EnablePostUsernameOverride: true,
+                EnablePostIconOverride: true,
+            },
+        };
+        cy.apiUpdateConfig(newSettings);
+
+        // # Login as sysadmin and ensure that teammate name display setting us set to default 'username'
+        cy.apiLogin('sysadmin');
+        cy.apiSaveTeammateNameDisplayPreference('username');
+
+        // # Create new team and create command on it
+        cy.apiCreateTeam('test-team', 'Test Team').then((teamResponse) => {
+            const team = teamResponse.body;
+            cy.visit(`/${team.name}`);
+
+            const webhookBaseUrl = Cypress.env().webhookBaseUrl;
```

Comment:
This jumped out as wrong to me, but I actually assume it should work? Cypress docs say to do `Cypress.env('foobar')` to grab variables, but without an argument it returns an object of all env vars which is why your approach works. No change required, just found it interesting!

Reply:
Yes, that works.  Can't remember where I first saw it but I like such approach (but my preference only).  We can change it or document on our style guide if we found it anti-pattern or something that degrades test performance.